fuzz_lightyear/fuzzer.py

```diff
@@ -105,11 +106,25 @@ def _fuzz_string(
     # https://swagger.io/docs/specification/data-models/data-types/#string
     kwargs = {}                                     # type: Dict[str, Any]
 
+    kwargs['min_size'] = parameter.get('minLength', 0)
+    kwargs['max_size'] = parameter.get('maxLength')
+
+    string_format = parameter.get('format')
+
+    if parameter.get('pattern'):
+        return st.from_regex(parameter['pattern'])
```

Comment:
(randomly clicked on this pr in slack) Super nit: Since this seems unrelated to the other checks/`kwargs`/`string_format`, we can maybe move it up above `kwargs =`